prow/config.yaml

```diff
@@ -144,6 +144,60 @@ presubmits:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  run_after_success:
+  - name: broker-coverage
+    agent: kubernetes
+    context: prow/broker-coverage.sh
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.2.1
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--clean"
+        - "--timeout=20"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+        - name: broker-codecov-token
+          mountPath: /etc/codecov
+          readOnly: true
+        - name: e2e-testing-kubeconfig
```

Comment:
Do your test need to run on a cluster ? If not please remove the e2e-testing-kubeconfig volume and volumeMount

Reply:
yes. It runs on cluster for crd configs.